docs/model_configuration.md

```diff
@@ -138,37 +138,47 @@ expected by the model.
 
 #### Special Conventions for PyTorch Backend
 
-***Naming Convention:*** Due to the absence of names for outputs in TorchScript
-models, the "name" attribute of outputs in the configuration must follow a
-specific naming convention i.e. "\<name\>__\<index\>". Where \<name\> can be
-any string and \<index\> refers to the position of the corresponding output.
-For inputs, the user has the choice to follow the above naming convention or use
-the names of input arguments to the forward function instead.
+**Naming Convention:** 
 
-This means that if there are two inputs and two outputs, the outputs must be named
-as: "OUTPUT__0" and "OUTPUT__1" such that "OUTPUT__0" refers to first output
-and "OUTPUT__1" refers to the second output.
+Due to the absence of sufficient metadata for inputs/outputs in TorchScript
+model files, the "name" attribute of inputs/outputs in the configuration must
+follow specific naming conventions. These are detailed below.
 
-Similarly, for the inputs, the first and second inputs would be named "INPUT__0"
-and "INPUT__1" respectively. Alternatively, if the forward function for the
-Torchscript model was defined as `forward(self, input0, input1)`, the first and
-second inputs could also be named "input0" and "input1" respectively. 
+1. [Only for Inputs] When the input is not a Dictionary of Tensors, the input
+names in the configuration file can mirror the names of the input arguements to
```

Comment:
Arguments, typo.

Reply:
Also, mixing of can and should. We need to be consistent about whether a user can do something or should/must do it.